Public/Src/Tools/Execution.Analyzer/Analyzers.Partner/PipExecutionPerformanceAnalyzer.cs

```diff
@@ -20,13 +20,19 @@ internal partial class Args
         public Analyzer InitializePipExecutionPerformanceAnalyzer()
         {
             string outputFilePath = null;
+            bool isCsvFormat = false;
             foreach (var opt in AnalyzerOptions)
             {
                 if (opt.Name.Equals("outputFile", StringComparison.OrdinalIgnoreCase) ||
                     opt.Name.Equals("o", StringComparison.OrdinalIgnoreCase))
                 {
                     outputFilePath = ParseSingletonPathOption(opt, outputFilePath);
                 }
+                else if (opt.Name.Equals("csv", StringComparison.OrdinalIgnoreCase) ||
+                    opt.Name.Equals("c", StringComparison.OrdinalIgnoreCase))
+                {
+                    isCsvFormat = true;

```

Comment:
nit: "csv" argument not only changes the format, but also the data included in the output file, so I'd use some kind of a  report name (i.e., describing the data) instead here. (rename both the arg and the variable)